docker/Dockerfile

```diff
@@ -36,8 +36,10 @@ RUN groupadd -r tigris && useradd -r -s /bin/false -g tigris tigris
 
 RUN mkdir -p /server /etc/tigrisdata/tigris /etc/foundationdb/
 
+ARG BUILD_PROFILE=""
```

Comment:
Isn't there already an `env` based mechanism for this somewhere?

Reply:
We need this at build time. Environment variables are available in container at runtime. For that reason we need build arg (i.e. build time env vars).

Let me know if I missed any existing mechanism to handle this.